server/_server/tests/models/test_org_chart.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+
+from server.models import Employee, OrgChart
+
+
+class TestOrgChart(TestCase):
+
+    def setUp(self):
+        jane = Employee('jane@tuenti.com')
```

Comment:
would be nice to add a test to assert number of minions of each supervisor, What do you think?

Reply:
cool, count on it